src/Nethermind/Nethermind.Serialization.Json/EthereumJsonSerializer.cs

```diff
@@ -20,18 +20,18 @@ public class EthereumJsonSerializer : IJsonSerializer
         private readonly int? _maxDepth;
         private readonly JsonSerializerOptions _jsonOptions;
 
```

Comment:
will revert all these.
Its code that spilled out from a failed attempt with the converter to see if it could be modified and somehow isolated